fimax/loans/doctype/gps_installation/gps_installation.js

```diff
@@ -2,7 +2,115 @@
 // For license information, please see license.txt
 
 frappe.ui.form.on('GPS Installation', {
-	refresh: function(frm) {
+	"onload": (frm) => {
+		if (frm.is_new()) {
+			frm.trigger("starts_date");
+		}
+	},
+	"starts_date": (frm) => {
```

Comment:
This field should be renamed to start_date (Start Date), but don't worry... I will do it later.